frame/proxy/src/lib.rs

```diff
@@ -137,84 +89,84 @@ pub struct Announcement<AccountId, Hash, BlockNumber> {
 	height: BlockNumber,
 }
 
-type CallHashOf<T> = <<T as Config>::CallHasher as Hash>::Output;
-
-decl_storage! {
-	trait Store for Module<T: Config> as Proxy {
-		/// The set of account proxies. Maps the account which has delegated to the accounts
-		/// which are being delegated to, together with the amount held on deposit.
-		pub Proxies get(fn proxies): map hasher(twox_64_concat) T::AccountId
-			=> (Vec<ProxyDefinition<T::AccountId, T::ProxyType, T::BlockNumber>>, BalanceOf<T>);
+#[frame_support::pallet]
+pub mod pallet {
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+	use super::{ *, DispatchResult };
```

Comment:
```suggestion
	use super::{*, DispatchResult};
```
Interesting, rust cannot disambiguate the same type.